shared/common-adapters/text.native.tsx

```diff
@@ -120,7 +120,7 @@ class Text extends React.Component<Props> {
           this._nativeText = ref
         }}
         selectable={this.props.selectable}
-        textBreakStrategy={this.props.textBreakStrategy}
+        textBreakStrategy={this.props.textBreakStrategy ?? 'simple'}
```

Comment:
this should be faster than the default on android and likely doesn't matter visually